templates/UmbracoPackage/.template.config/dotnetcli.host.json

```diff
@@ -2,17 +2,21 @@
   "$schema": "https://json.schemastore.org/dotnetcli.host.json",
   "symbolInfo": {
     "Framework": {
-      "longName": "Framework",
-      "shortName": "F",
+      "longName": "framework",
+      "shortName": "f",
       "isHidden": true
     },
     "UmbracoVersion": {
       "longName": "version",
-      "shortName": "v"
+      "shortName": ""
     },
```

Comment:
These changes have already been discussed in https://github.com/umbraco/Umbraco-CMS/pull/13776#discussion_r1094552556 and we shouldn't keep pushing this change back (especially since we only allow a single framework version anyway)...